nixos/modules/services/networking/syncthing.nix

```diff
@@ -484,6 +484,24 @@ in {
               -gui-address=${cfg.guiAddress} \
               -home=${cfg.configDir}
           '';
+          MemoryDenyWriteExecute = true;
+          NoNewPrivileges = true;
+          PrivateDevices = true;
+          PrivateMounts = true;
+          PrivateTmp = true;
+          PrivateUsers = true;
+          ProtectControlGroups = true;
+          ProtectHostname = true;
+          ProtectKernelModules = true;
+          ProtectKernelTunables = true;
+          RestrictNamespaces = true;
+          RestrictRealtime = true;
+          RestrictSUIDSGID = true;
+          CapabilityBoundingSet = [
```

Comment:
Can we invert this list, by only listing the caps we need?

http://0pointer.de/blog/projects/security.html does it that way, and I'm not sure why we'd need it the other way round.

Reply:
There are ~40 capabilities.

While I'm pretty sure we don't want to use some, I'm unsure about some other and fear I'm going to break the service in unexpected ways.

Since the test coverage here is close to 0, I'd like to start collecting the low hanging fruit by blocking the obvious worst offenders here.